client/analytics/components/report-chart/test/index.js

```diff
@@ -1,45 +1,9 @@
-/**
- * External dependencies
- */
-import { mount } from 'enzyme';
-
 /**
  * Internal dependencies
  */
-import { ReportChart } from '../';
 import { getChartMode, getSelectedFilter } from '../utils';
 
-const path = '/analytics/revenue';
-const data = {
-	data: {
-		intervals: [],
-	},
-	isEmpty: false,
-	isError: false,
-	isRequesting: false,
-};
-const selectedChart = {
-	key: 'total_sales',
-	label: 'Total Sales',
-	type: 'currency',
-};
-
 describe( 'ReportChart', () => {
-	test( 'should set the time-comparison mode prop by default', () => {
-		const reportChart = mount(
-			<ReportChart
-				path={ path }
-				primaryData={ data }
-				query={ {} }
-				secondaryData={ data }
-				selectedChart={ selectedChart }
-			/>
-		);
-		const chart = reportChart.find( 'Chart' );
-
-		expect( chart.props().mode ).toEqual( 'time-comparison' );
-	} );
```

Comment:
I believe that DOM changes would result from this test, but they would be in the nested `<Chart>` component, not the `<ReportChart>` that is under test here.

I removed this test case because it felt really low value. It only covers the default case here: https://github.com/woocommerce/woocommerce-admin/blob/7232d514329e54e61711fb3d09d8cbfceb9f356d/client/analytics/components/report-chart/index.js#L355-L358